packages/shared/lib/core/profile/actions/profiles/checkAndMigrateProfiles.ts

```diff
@@ -156,6 +156,9 @@ function persistedProfileMigrationToV9(existingProfile: IPersistedProfile): void
 }
 
 function persistedProfileMigrationToV10(existingProfile: IPersistedProfile): void {
+    const network = NETWORK?.[existingProfile?.networkProtocol]?.[existingProfile?.networkType]
+    existingProfile.network = network
+
```

Comment:
It looks like this same code snippet is in the `...ToV11` migration function too, should we check that the `network` isn't `undefined` before updating the profile, or rather check if that `network` property already exists on the profile and if it does then don't touch it?

Reply:
must have missed it will move to v11